### ISSUE TITLE
Error on Install for v13.0.1 | Windows 10 ver. 1909, build 18363.476 | no OS support

### ISSUE THREAD

**Issue by richard-stanton** [ShareX/ShareX#4509]
I am a Windows Insider, however this build and version are on GA for anyone who clicks the update button. 

**Error on in install:** 
> This program does not support the version of Windows your computer is running.

**Version of Windows:** 
Windows 10 Professional, v1909, build 18363.476
**Steps to reproduce:** 
1. Go to **System** >  **Home** (if not already there) > **Update & Security** > **Windows Update**
2. Click "**Check for updates**" (may need to do it twice to kick off update)
3. Allow Windows Update to upgrade you to v1909
4. Attempt to install ShareX v13.0.1 installer version from releases: 
[https://github.com/ShareX/ShareX/releases/download/v13.0.1/ShareX-13.0.1-setup.exe](url)

**Note:** This appears to only impact the wrapper code for the installer version, portable version is working as expected.

**Comment by Jaex** [comment]
I'm also using "Windows 10 Professional, v1909, build 18363.476" so not sure why it not works in your OS.

Maybe it is related to InnoSetup version we were using. Recently we updated it to last version, so next version released gonna use that one. Which can maybe fix your issue.

**Comment by richard-stanton** [comment]
After trying on a 2nd and 3rd machine same build and similar config the installer worked. Coming back to the original machine all three of the test binaries I downloaded at the time which were failing were able to progress farther than before and allowed for installation. Nothing changed on the machine other than downloading and using the portable version. I did have Glasswire running, but in passive mode (both times), but I have to assume it was some sort of interference from an overly aggressive AV process possibly blocking the install (though I have not disabled anything). Regardless I can't explain it, and I cannot reproduce it any more so I think this report can be classified CNR for now. If I am able to trigger it again and is reproducible I will report back my findings. Thanks for the info on your end, it spurred me to try a few more machines when available. :)
